backend/src/types.ts

```diff
@@ -613,6 +613,19 @@ export type PrometheusResponse = {
   status: string;
 };
 
+export type PrometheusQueryRangeResponse = {
+  data: {
+    result: [
+      {
+        metric: unknown;
```

Comment:
Do we not know what this is? Do we not need/care about it?

Reply:
Based on the official doc, it should be something like
```
 "metric" : {
  "__name__" : "up",
  "job" : "prometheus",
  "instance" : "localhost:9090"
},
```
I can set it to `Record<string, string>` thought... but I don't think we need to care about it. It's nothing related to the data.

Reply:
Ah okay -- not sure where the official docs are, are they linkable? Either way, I'd like some comment to say "not needed".

Shape definition is important, but props we don't need don't hurt to be defined as `unknown` -- I just don't want us to cut corners and make bad / incomplete types

Reply:
https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries sure, it's here.

Reply:
Yeah, so something like:
```suggestion
        // not used -- see https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries for more info
        metric: unknown;
```